pkg/zui.go

```diff
@@ -0,0 +1,7 @@
+package pkg
+
+//go:generate zbusc -module zui -version 0.0.1 -name zui -package stubs github.com/threefoldtech/zos/pkg+ZUI stubs/zui_stub.go
+
+type ZUI interface {
+	PushErrors(label string, errors []string)
```

Comment:
```suggestion
	PushErrors(label string, errors []string) error
```

Just to make the linter happy about the genreate stubs